website/blog/building-a-fast-trends-page.md

```diff
@@ -0,0 +1,80 @@
+---
+slug: building-a-fast-trends-page.md
+title: "How We Use ClickHouse to Analyze Trends Across Millions of Builds"
+author: Lulu Zhang
+author_title: Engineer @ BuildBuddy
+date: 2022-11-18:12:00:00
+author_url: https://www.linkedin.com/in/luluzhang66
+author_image_url: https://avatars.githubusercontent.com/u/3977083?v=4
+image: /img/blog/clickhouse.png
+tags: [product, engineering]
+---
+
+When you use Buildbuddy with Bazel to build and test software, Buildbuddy
+captures information about each Bazel invocation, such as number of builds,build
+duration, remote cache performance, and more. Buildbuddy has
+a [Trends](https://app.buildbuddy.io/trends/) page to visualize trends in this
+data over time.
+
+![](/img/blog/trends-2.png)
+
+The trends page allows you to see how improvements you are making to your builds
+affects your average build duration and other stats. It also exposes areas that
+might need improving. For example, if you see the cache hit rate go down over
+time, your build might have some non-deterministic build actions that could be
+improved, or some newly introduced dependencies in your build that result in
```

Comment:
nit: I think you can remove `in your build`

`your build might have some` covers the second clause just fine

Reply:
Done